src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +56,40 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        return dateParams.length == 0 ? Optional.empty() : dateParams[1] > 12 ? Optional.empty() :
+            Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]));
```

Comment:
why do you make checking only for month? (>12)?
in this way, we can add also a checking for days (>31) or additional checking for february and other less than 31 days months :D

Reply:
fixed